core/embed/rust/src/ui/model_mercury/component/keyboard/bip39.rs

```diff
@@ -161,7 +161,7 @@ impl Component for Bip39Input {
         let style = self.button.style();
 
         // First, paint the button background.
-        self.button.render_background(target, style);
+        self.button.render_background(target, style, 0xFF);
```

Comment:
Note: this line was removed in https://github.com/trezor/trezor-firmware/pull/3832 so it will cause conflict later.